code42_connector.py

```diff
@@ -40,6 +64,17 @@ def _convert_to_obj_list(scalar_list, sub_object_key="item"):
     return [{sub_object_key: item} for item in scalar_list]
 
 
+def add_eq_filter(filters, value, filter_class):
+    return filters.append(filter_class.eq(value))
+
+
+def is_default_dict(_dict):
+    for key in _dict:
+        if _dict.get(key):
```

Comment:
Would `{"foo": False}` be a false positive here?

Reply:
`False` would be the default for any checkbox item. Currently the only one of these is `untrusted_only`. If that's unchecked, and there is nothing else set, that means "get all the events", which is exactly what we want to avoid -- so in the current state of the app there's nothing wrong with this behavior.

Reply:
You can change the default options I am pretty sure. I see what you mean though, regarding trusted_activity being the only one and wanting to avoid getting all the results...

I think it is probably fine for now, but it would be better to be able instead check that each individual parameter is not its own default. I am not sure if the input schema contains that info or not or if there is any other way to know.